src/connectors/menu/__tests__/connectMenu-test.js

```diff
@@ -2,7 +2,13 @@ import jsHelper, {
   SearchResults,
   SearchParameters,
 } from 'algoliasearch-helper';
+import algoliasearchHelper from 'algoliasearch-helper/dist/algoliasearch.helper';
```

Comment:
You don't need this. `jsHelper` on the first line is the same one.

Reply:
Fixed